_posts/guides/2020-09-05-guide_custom-hud-menus.md

```diff
@@ -32,59 +32,64 @@ To open a hud menu, use [`{{ page.ccom_show }}`](/command/{{ page.ccom_show }}).
 The parameter for this command will autofill with all the available hud menus.
 
 ## File reading/writing
-Hud menus are stored in and read from `cfg/menus/` as `.vdf` files.
+Hud menus are stored in and read from `cfg/menus/` as `.kv3` files.
+Hud menus utilize Valve developed data format known as KeyValues3 (KV3).
 Files with the suffix `_default` are not read from unless their counterpart does not exist. 
 In which case, it will be copied and have the suffix removed for reading.
 
 ## Custom hud menus how-to
 A simple hud menu with just one command looks like:
 ```
-"Custom Menu"
+<!-- kv3 encoding:text:version{e21c7f3c-8a33-41c5-9977-a76d3a32aa0d} format:generic:version{7412167c-06e9-4698-aff2-e63eb59037e7} -->
 {
-  "1"
-  {
-    "label"   "Some command"
-    "command" "mom_some_command"
-  }
+  main_commands =
+  [
+    {
+      "label"   "Some command"
+      "command" "mom_some_command"
+    }
+  ]
 }
 ```
 `"1"` refers to the number in the menu the command is at, `"label"` is the text that shows for the button on the menu, and `"command"` is the console command that's inputted into console.
 
 Adding more commands is then as simple as:
 ```
-"Custom Menu"
+<!-- kv3 encoding:text:version{e21c7f3c-8a33-41c5-9977-a76d3a32aa0d} format:generic:version{7412167c-06e9-4698-aff2-e63eb59037e7} -->
 {
-  "1"
-  {
-    "label"   "Some command"
-    "command" "mom_some_command"
-  }
-  "2"
-  {
-    "label"   "Some other command"
-    "command" "mom_some_other_command"
-  }
-  "3"
-  {
-    "label"   "Some command"
-    "command" "mom_some_other_command"
-  }
-  ...
+  main_commands =
+  [
+    {
+      "label"   "Some command"
+      "command" "mom_some_command"
+    },
+    {
+      "label"   "Some other command"
+      "command" "mom_some_other_command"
+    },
+    {
+      "label"   "Some other command"
+      "command" "mom_some_other_command"
+    },
+    ...
+  ]
 }
 ```
 As in this example, labels and commands can be duplicated.
 Hud menus allow a maximum of 9 commands, accessed through numbers 1-9.
```

Comment:
This is no longer accurate either, the menu will paginate accordingly when you add more than 9 entries